internal/jobs/noop_job.go

```diff
@@ -19,6 +20,9 @@ var NoOperationFailure = errors.New("job failed on request")
 
 // Unmarshall arguments and handle error
 func HandleNoop(ctx context.Context, job *worker.Job) {
+	ctx, span := otel.Tracer(TraceName).Start(ctx, "HandleNoop")
```

Comment:
nit
Why are you using a global trace name and not s specific one regarding the noop job?